app/models/game.rb

```diff
@@ -254,12 +210,12 @@ def save_state state
     self.state = state.to_yaml
   end
 
-  def get_highest_card cards, first_suit_played, trump, start_index
+  def get_highest_card cards, first_suit_played, trump
     return cards.first if cards.size <= 1
 
     # if trump was played, ignore other cards
     # an ace indicates no trump, so ignore it if that's the case
-    trump_cards = (trump.suit =~ /ace/i) ? [] : cards.select { |c| c.suit == trump.suit }
+    trump_cards = (trump.value == 12) ? [] : cards.select { |c| c.suit == trump.suit }
```

Comment:
yeah I mean it looks cool but its totally unnecessary. I'll change this to use value_name